python/ray/experimental/channel/nccl_group.py

```diff
@@ -307,3 +304,6 @@ def destroy(self) -> None:
             # flag is True when they exit from the abort.
             self._comm.abort()
             self._comm.destroy()
+
+    def get_device_type(self) -> str:
+        return "nccl"
```

Comment:
```suggestion
        return "gpu"
```

"nccl" is the transport name but "gpu" is the device that each actor is expected to have.

You could add a `get_transport_name` and that can return NCCL instead.